lib/app/css/components/github.css

```diff
@@ -62,6 +63,7 @@ github.com style (c) Vasily Polovnyov <vast@whiteants.net>
 
 .hljs-class .hljs-title,
 .hljs-type,
+.hljs-attr,
```

Comment:
I added these selectors so that highlight colors remain the same after the upgrade. Please let me know if there are any styles that don't seem correct to you.